File: csharp/main/ImportNamespaceDef.cs
using JetBrains.Annotations;

namespace Deltix.Luminary
{
	public class ImportNamespaceDef : ImportDef
	{
		public ImportNamespaceDef([NotNull] NamespaceDef target)
			: base(ImportKind.Namespace)
		{
			Target = target;
		}

		[NotNull]
		public NamespaceDef Target { get; }
	}
}

File: java/src/main/java/deltix/luminary/ImportNamespaceDef.java
package deltix.luminary;

import org.jetbrains.annotations.NotNull;

public class ImportNamespaceDef extends ImportDef {
    private final NamespaceDef target;

    public ImportNamespaceDef(@NotNull NamespaceDef target) {
        super(ImportKind.NAMESPACE);
        this.target = target;
    }

    @NotNull
    public NamespaceDef getTarget() {
        return target;
    }
}
